packages/components/package.json

```diff
@@ -76,7 +76,7 @@
     "vis-network": "~6.5.2"
   },
   "devDependencies": {
-    "@labkey/build": "6.2.0",
+    "@labkey/build": "6.2.1",
```

Comment:
were you going to bump the components package itself as well?

Reply:
As we discussed, I did not publish a new labkey-ui-components package for this change